### ISSUE TITLE
event name generation generates name that's not useable in twitter search

### ISSUE THREAD

**Issue by matthewberryman** [MSFREACH/msf-reach#89]

**Comment by LucieGueuning** [comment]
as per discussion, it is now based on imagination of users. Which is not a permanent solution. We need to have type and location as keyword (at least) and then have a good list of tweet based on trusted profile (#275)

**Comment by LucieGueuning** [comment]
SOLUTION: If user is not putting info on the field when creation, the system allocate automatically the Event Name as follow: Type + Location + Time. This field stay editable.

**Comment by matthewberryman** [comment]
@LucieGueuning this is what I have now with a change I made (sorry if it's not quite right - the combination of your comments is a little bit unclear to me):
If the event has a name, then the search terms are name and country. (In testing I found that adding the country helps a lot).
If the event doesn't have a name, then the search terms are type + sub_type (if there's a sub_type) bearing in mind that type and subtype will change in future #30 to be a list of more than one if selected + event date and time (with change already made for #30 time entry).

It's really easy to change this code - sing out if you want it different / have any ideas.
